recipes/sentry-native/all/conanfile.py

```diff
@@ -130,6 +130,8 @@ def validate(self):
             raise ConanInvalidConfiguration("This version doesn't support ARM compilation")
 
     def build_requirements(self):
+        if tools.Version(self.version) >= "0.4.1":
```

Comment:
```suggestion
        if self.settings.os == "Windows" and tools.Version(self.version) >= "0.4.0":
```

Reply:
https://github.com/getsentry/sentry-native/blob/0.4.0/CMakeLists.txt#L1-L8

Reply:
Honestly, it should be replaced upstream by:

```cmake
cmake_minimum_required(VERSION 3.10)
if(POLICY CMP0091)
    cmake_policy(SET CMP0091 NEW)
endif()
```

Reply:
I have no idea what the issue is with 0.4.1... Is it an option to drop support for old version ? This PR aims only to add add the latest version to the CCI.